package.json

```diff
@@ -83,6 +83,7 @@
   },
   "homepage": "https://github.com/lijiarui/wechaty-puppet-padchat#readme",
   "dependencies": {
+    "@babel/runtime": "7.0.0-beta.39",
```

Comment:
You should also consider to add this dependenciues for a dirty fix or workaround on v0.8 because currently it is broken for all users.

Reply:
Got it, will do. 

ps: Should have moved to your version of `json-rpc-peer` early